src/config.rs

```diff
@@ -79,7 +83,15 @@ pub struct Config {
     pub(crate) target: LevelFilter,
     pub(crate) target_padding: TargetPadding,
     pub(crate) location: LevelFilter,
+    #[cfg(feature = "chrono")]
+    pub(crate) time_format: Cow<'static, str>,
```

Comment:
Could we maybe not have totally different types for the two features?
As a suggestion we could just change the `Custom` variant of the `TimeFormat` enum to contain a 'static Cow, when chrono is enabled and still support Rfc2822 and Rfc3339 out of the box with both variants.